bin/aurbuild

```diff
@@ -133,6 +133,10 @@ if [[ $build == build_chroot ]]; then
     sudo cp -v "$tmp"/pacman.conf "$CHROOT"/root/etc/pacman.conf
 fi
 
+# Canonicalise paths
+pool="$(realpath "${pool}")"
+root="$(realpath "${root}")"
```

Comment:
The error checking above after getopts should already ensure this doesn't error — we *know* the path exists, we just need the full path and not the relative path.